variables.tf

```diff
@@ -74,7 +74,7 @@ variable "tfc_agent_log_level" {
 
 variable "tfc_agent_data_dir" {
   type        = string
-  default     = "~/.tfc-agent"
+  default     = ""
```

Comment:
Wondering if this shouldn't be null instead.

Reply:
Makes sense to me.